src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -259,9 +273,21 @@ public void eventsB_actionPerformed(ActionEvent e) {
 	}
 
 	public void filesB_actionPerformed(ActionEvent e) {
-		cardLayout1.show(panel, "FILES");
-		setCurrentButton(filesB);
-		Context.put("CURRENT_PANEL", "FILES");
+		if (Configuration.get("RESOURCE_PATH").equals("nothing")) {
```

Comment:
this doesn't account for existing users, whose properties files will not have a `RESOURCE_PATH` entry